recipe/meta.yaml

```diff
@@ -1,55 +1,321 @@
 {% set name = "google-api-core" %}
-{% set version = "1.8.2" %}
-{% set sha256 = "16b14c0492154110d869a08be22d27440010057fd826205303c7edaa1c3cd71e" %}
-{% set build = 0 %}
+{% set version = "1.9.0" %}
+{% set sha256 = "fc1fea74bd863fb71486066e0c6b3a4dad26fb70ec61a0edcada8637feb77c68" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name|lower }}-feedstock
```

Comment:
We're getting some weird `feedstock-feedstock` text in the readme with this change. Why the package name switch?

Reply:
Previously this repo only produced 1 package, now there are 3, 4 if you include the root which doesn't actually produce a package.  Some of the other multi-output projects are using {{ name }}-split instead of {{ name }}-feedstock.

The only thing that is set in stone is the root package name can't be `google-api-core` because that has to be the package name of one of the outputs.

Thoughts?

Reply:
`{{ name }}-split` sounds good to me, thanks.